lib/model.js

```diff
@@ -558,7 +558,7 @@ Model.prototype.hasAndBelongsToMany = function(joinedModel, fieldDoc, leftKey, r
 
   var linkModel;
   if (thinky.models[link] === undefined) {
-    linkModel = thinky.createModel(link, {}); // Create a model, claim the namespace and create the table
+    linkModel = thinky.createModel(link, {}, { table: self._table }); // Create a model, claim the namespace and create the table
```

Comment:
`self._table` is simply the table settings (e.g. number of replicas and shards) object for the current model. I'm just passing the table settings along to the model that will inevitably create the n-n relation.